electron_app/src/ipc/manager.js

```diff
@@ -119,11 +119,12 @@ ipc.answerRenderer('fs-delete-email-content', async params => {
 });
 
 ipc.answerRenderer('db-create-email', async params => {
+  const [account] = await dbManager.getAccountByParams({id: params.accountId})
   await fileUtils.saveEmailBody({
     body: params.body,
     headers: params.headers,
     metadataKey: parseInt(params.email.key),
-    username: getUsername()
+    username: account.recipientId.indexOf('@') > -1 ? account : `${account.recipientId}@${APP_DOMAIN}` 
```

Comment:
If you prefer, you can use `account.recipientId.includes('@')` instead `indexOf()`